etc/HIReplayOfflineConfiguration.py

```diff
@@ -1337,7 +1373,7 @@
                write_dqm=True,
                alca_producers=["TkAlMinBias"],
                dqm_sequences=["@common"],
-               scenario=hiTestppScenario)
+               scenario=hiLightEventsScenario)
```

Comment:
@LinaresToine I don't understand this choice (and also several others).
In 2023 this PD was processed with `ppEra_Run3_pp_on_PbPb_2023`. I think this should be `hiHeavyEventsScenario` (= `"ppEra_Run3_pp_on_PbPb_2024"`) now.